src/core/utils.ts

```diff
@@ -113,15 +114,26 @@ export function getSeriesMarkerType(series?: Highcharts.Series): ChartSeriesMark
   }
 }
 
-// Highcharts supports color objects to represent gradients and more, but we only support string
-// colors in our markers. In case a non-string color is defined, we use black color as as fallback.
-export function getSeriesColor(series?: Highcharts.Series): string {
-  return typeof series?.color === "string" ? series.color : "black";
-}
 export function getPointColor(point?: Highcharts.Point): string {
   return typeof point?.color === "string" ? point.color : "black";
 }
 
+/**
+ * Gets the appropriate color for a chart series based on its type.
+ * @param chart - The Highcharts chart object
+ * @param index - The index of the series in the chart
+ * @returns The color string for the series, or "black" as fallback
+ */
+export function getColor(chart: Highcharts.Chart, index: number): string {
```

Comment:
The index is always seriesIndex, or? I would recommend calling it seriesIndex in that case for clarity. Likewise, the util is better be called getSeriesColor to avoid confusion, and for parity with getPointColor.